test.go

```diff
@@ -24,8 +24,11 @@ func newTestRunner(o *operator) (*testRunner, error) {
 	}, nil
 }
 
-func (rnr *testRunner) Run(ctx context.Context, cond string) error {
+func (rnr *testRunner) Run(ctx context.Context, cond string, runned bool) error {
 	store := rnr.operator.store.toMap()
+	if runned {
+		store[storeCurrentKey] = rnr.operator.store.latest()
+	}
```

Comment:
I see, indeed, that’s right

So you are enabling the current variable only after execution?

Reply:
👍 I think this was the only point in time to use a current variable.